docs/layouts/shortcodes/generated/forst_configurable_configuration.html

```diff
@@ -0,0 +1,168 @@
+            <td>The maximum number of open files (per stateful operator) that can be used by the DB, '-1' means no limit. The default value is '-1'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.log.dir</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The directory for ForSt's information logging files. If empty (Flink default setting), log files will be in the same directory as the Flink log. If non-empty, this directory will be used and the data directory's absolute path will be used as the prefix of the log file name. If setting this option as a non-existing location, e.g '/dev/null', ForSt will then create the log under its own database folder as before.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.log.file-num</h5></td>
+            <td style="word-wrap: break-word;">4</td>
+            <td>Integer</td>
+            <td>The maximum number of files ForSt should keep for information logging (Default setting: 4).</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.log.level</h5></td>
+            <td style="word-wrap: break-word;">INFO_LEVEL</td>
+            <td><p>Enum</p></td>
+            <td>The specified information logging level for ForSt. If unset, Flink will use <code class="highlighter-rouge">INFO_LEVEL</code>.<br />Note: ForSt info logs will not be written to the TaskManager logs and there is no rolling strategy, unless you configure <code class="highlighter-rouge">state.backend.forst.log.dir</code>, <code class="highlighter-rouge">state.backend.forst.log.max-file-size</code>, and <code class="highlighter-rouge">state.backend.forst.log.file-num</code> accordingly. Without a rolling strategy, long-running tasks may lead to uncontrolled disk space usage if configured with increased log levels!<br />There is no need to modify the ForSt log level, unless for troubleshooting ForSt.<br /><br />Possible values:<ul><li>"DEBUG_LEVEL"</li><li>"INFO_LEVEL"</li><li>"WARN_LEVEL"</li><li>"ERROR_LEVEL"</li><li>"FATAL_LEVEL"</li><li>"HEADER_LEVEL"</li><li>"NUM_INFO_LOG_LEVELS"</li></ul></td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.log.max-file-size</h5></td>
+            <td style="word-wrap: break-word;">25 mb</td>
+            <td>MemorySize</td>
+            <td>The maximum size of ForSt's file used for information logging. If the log files becomes larger than this, a new file will be created. If 0, all logs will be written to one log file. The default maximum file size is '25MB'. </td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.rescaling.use-delete-files-in-range</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>If true, during rescaling, the deleteFilesInRange API will be invoked to clean up the useless files so that local disk space can be reclaimed more promptly.</td>
```

Comment:
what are the `useless files` in this scenario?

Reply:
actually, it is not `useless files` but `useless key-values`. Changed